mods/beds/locale/beds.ru.tr

```diff
@@ -0,0 +1,8 @@
+# textdomain: beds
+Fancy Bed=Детализированная кровать
+Simple Bed=Обычная кровать
+Leave Bed=Покинуть кровать
+Good morning.=Доброе утро
```

Comment:
full sentences should generally also end with `.` when translated

Reply:
What for? No one of the original ends with '.'